packages/web-vite/src/router/AppRouter.tsx

```
@@ -87,25 +88,31 @@ const AppLayout: React.FC = () => {
+      <div className="main-content-wrapper">
+        {/* Top bar with user actions */}
+        <header className="top-bar">
+          <div className="top-bar-left">
+            {/* Search will be in individual pages */}
+          </div>
+          <div className="top-bar-right">
+            <a href="/settings" className="top-bar-link">
+              ⚙️ Settings
+            </a>
+            <button className="logout-btn" onClick={handleLogout}>
+              Logout
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)